conbench/app/_plots.py

```diff
@@ -175,6 +174,7 @@ def _insert_nans(some_list: list, indexes: List[int]):
     return some_list
 
 
+@no_type_check
```

Comment:
I tried my luck to add a TypeDict for the `data` arg. But backed out of this.


https://adamj.eu/tech/2021/05/10/python-type-hints-how-to-use-typeddict/